### PR TITLE
Feature/csh/samples

### PR DESCRIPTION
**Pull request by cshcomcom**
## Description
- Add samples/event_manager.js file
- Update Ain.eh README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -271,6 +271,20 @@ const accounts = ain.db.ref('/accounts').getValue().then(result => {
 `static fromPrivateKey(privateKey: Buffer): Account`
 - Imports an account from a private key.
 
+### Ain.eh
+`connect(connectionOption?: EventChannelConnectionOption, disconnectCallback?: DisconnectCallback)`
+- Connect to event handler node. Must be called before subscribing.
+
+`disconnect()`
+- Disconnect from the event handler node and unsubscribe from all subscriptions.
```
Comment (platfowner) [inline README.md:279]: s/unsubscribe from/unsubscribe/?

**Review by platfowner — APPROVED** [review]
LGTM with some comments.
Thanks!

File: README.md (hunk 1 above)
Comment (cshcomcom) [reply README.md]: Looks better! Done

**Comment by cshcomcom** [comment]
Thanks for the quick review! Now merging

**Pull request merged**